server/session/session.go

```diff
@@ -439,6 +439,7 @@ func (s *Session) registerHandlers() {
 		packet.IDPlayerAction:          &PlayerActionHandler{},
 		packet.IDPlayerAuthInput:       &PlayerAuthInputHandler{},
 		packet.IDPlayerSkin:            &PlayerSkinHandler{},
+		packet.IDRequestAbility:        nil, // TODO: Properly support this.
```

Comment:
Think you can remove the TODO for this, we don't really need to handle this at any point in the near future.

Reply:
Removed.